scipy/interpolate/_ndgriddata.py

```diff
@@ -92,7 +92,7 @@ class NearestNDInterpolator(NDInterpolatorBase):
     def __init__(self, x, y, rescale=False, tree_options=None):
         NDInterpolatorBase.__init__(self, x, y, rescale=rescale,
                                     need_contiguous=False,
-                                    need_values=False)
+                                    need_values=True)
```

Comment:
What is the effect of the change of the default value and is this change really needed?

Reply:
This is a residue of a discarded change, replacing with original value.